scripts/js/pw/startHatchingOriginOfShells.js

```diff
@@ -0,0 +1,90 @@
+require('dotenv').config();
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+const { waitExtrinsicFinished, setStatusType, getNonce, token, waitTxAccepted} = require('./pwUtils');
+
+const alicePrivkey = process.env.ROOT_PRIVKEY;
+const bobPrivkey = process.env.USER_PRIVKEY;
+const overlordPrivkey = process.env.OVERLORD_PRIVKEY;
+const ferdiePrivkey = process.env.FERDIE_PRIVKEY;
+const charliePrivkey = process.env.CHARLIE_PRIVKEY;
+const davidPrivkey = process.env.DAVID_PRIVKEY;
+const evePrivkey = process.env.EVE_PRIVKEY;
+const endpoint = process.env.ENDPOINT;
+
+async function setIncubationProcess(khalaApi, overlord, status) {
+    let nonceOverlord = await getNonce(khalaApi, overlord.address);
+    return new Promise(async (resolve) => {
+        console.log(`Setting CanStartIncubationStatus to ${status}...`);
+        const unsub = await khalaApi.tx.pwIncubation.setCanStartIncubationStatus(status
+        ).signAndSend(overlord, {nonce: nonceOverlord++}, (result) => {
+            if (result.status.isInBlock) {
+                console.log(`Transaction included at blockHash ${result.status.asInBlock}`);
+            } else if (result.status.isFinalized) {
+                console.log(`Transaction finalized at blockHash ${result.status.asFinalized}`);
+                unsub();
+                resolve();
+            }
+        });
+        await waitTxAccepted(khalaApi, overlord.address, nonceOverlord - 1);
+        console.log(`Enabling the Incubation Process...Done`);
+    });
+}
+
+async function hatchOriginOfShell(khalaApi, overlord, originOfShellsOwners) {
+    const defaultMetadata = "";
```

Comment:
Is it `ar://` for the final NFT?

Reply:
Yeah, this is the default NFT for the Shell NFT. Something that can be used as the default Shell NFT if the Shell NFT isn't equipped with the appropriate number of Shell parts. Not sure if this is needed though.